src/sass/app_2/_common/footer.scss

```diff
@@ -1,7 +1,45 @@
 /* stylelint-disable color-no-hex */
 footer {
-    padding: 10px 0;
+    position: fixed;
+    bottom: 0px;
+    padding: 5px 10px;
     width: 100%;
-    box-shadow: 0 2px 8px #f0f1f2;
+    border-top: 1px solid #e9e9e9;
+    background-color: #fff;
+    box-sizing: border-box;
+
```

Comment:
We probably should add `z-index: 1` as there are some elements like symbols which are `absolute` and might float in front of the footer once it overlaps.